DSA-Campaign-Uplift-Estimation/src/main/webapp/Home/script-home.js

```diff
@@ -10,4 +10,143 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+// Sets the page to always display the first two elements when loading the page.
+var secondChart = 2;
+var firstChart = 0;
+
+google.charts.load('current', {'packages':['bar', 'table']});
+
+// Retrieves a list of avaliable keyword campaigns the user has stored and is used in
+// dropdown menu.
+function getKeywordCampaigns() {
+	const keywordCampaignList = document.getElementById('keyword-campaigns');
+    keywordCampaignList.innerHTML = '<option value=0>Select a keyword campaign</option>';
+
+    fetch('/keyword-campaigns').then(response => response.json()).then(keywordCampaigns => {
+        keywordCampaigns.forEach(keywordCampaign => {
+            keywordCampaignList.innerHTML += '<option value=' + keywordCampaign.keywordCampaignId + '>' + keywordCampaign.name  + '</option>';
+        });
+    });
+}
+
+// Checks if user has selected a keyword campaign. If the user has selected a keyword
+// then the for loop draws 2 graphs and creates pagination to explore other charts.
+function getDSACampaigns() {
+    const DSACampaignsList = document.getElementById('DSA-campaigns');
+    DSACampaignsList.innerHTML = '<p>First select a keyword campaign.</p>';
+    var keywordCampaignId = document.getElementById("keyword-campaigns").value;
+
+    if (keywordCampaignId != 0) {
+        DSACampaignsList.innerHTML = '';
+
+        fetch('/DSA-campaigns?keywordCampaignId=' + keywordCampaignId).then(response => response.json()).then(DSACampaigns => {
+
+            for(var i = firstChart; i < secondChart; i++ ){
```

Comment:
why not make this dynamic? It is a little strange to have the loop of two charts in this manner, because it implies that firstChart and secondChart may not be one apart.

Reply:
Fixed.